subsys/bluetooth/audio/ascs.c

```diff
@@ -1593,7 +1593,7 @@ static void ase_qos(struct bt_ascs_ase *ase, const struct bt_ascs_qos *qos)
 			} else if (cqos->pd == 0) {
 				reason = BT_ASCS_REASON_PD;
 			}
-		} else if (err == -EADDRINUSE) {
+		} else if (err == -EADDRINUSE || err == -EALREADY) {
```

Comment:
Hmm, I'm wondering if `ase_stream_qos` should instead just return `EADDRINUSE`? Just did a quick search, and `EADDRINUSE` is not return anywhere in ASCS, so it probably return `EADDRINUSE` at some point. 

In any case, we should either change the return value of `ase_stream_qos` to `EADDRINUSE` or change this line here to check for `EALREADY` instead of `EADDRINUSE`; there isn't any reason to check for both here. 

I suggest to just replace `EADDRINUSE` with `EALREADY` in this line here
```suggestion
		} else if (err == -EALREADY) {
```